core/src/com/unciv/ui/worldscreen/AlertPopup.kt

```diff
@@ -63,6 +63,15 @@ class AlertPopup(val worldScreen: WorldScreen, val popupAlert: PopupAlert): Popu
                     close()
                 })
             }
+            AlertType.BorderConflict -> {
+                val translatedNation = worldScreen.gameInfo.getCivilization(popupAlert.value).getTranslatedNation()
+                addLeaderName(translatedNation)
+                addGoodSizedLabel("Remove your troops in our border immediately!").row()
+                val responseTable = Table()
+                responseTable.add(getCloseButton("Sorry."))
```

Comment:
This should not decrease influence, but should add a 'agreed to remove troops' countdown - so if you _don't_ remove the troops, you'll get a different popup of 'your troops remain in our territory, we'll remember this', with a bigger influence decrease than if you had outright disagreed

Reply:
Why we need two relationship damaging solutions for this? In original civ5 it is as simple as "Retreat your troop or suffer penalty."